src/Components/Admin/EnvironmentEditor/environment.js

```diff
@@ -19,6 +47,26 @@ export default function EnvironmentEditor() {
             <div className={classes.page}>
                 <EnvironmentBar />
             </div>
+            <div className={classes.container}>
+                <div className={classes.emptyButtonsContainer}>
+                    <div className={classes.buttonContainer}>
+                        <Button
+                            startIcon={<AddIcon />}
+                            className={classes.button}
+                        >
+                            New Scene from Scratch
+                        </Button>
+                    </div>
+                    <div className={classes.buttonContainer}>
+                        <Button
+                            startIcon={<AccountBalanceIcon />}
+                            className={classes.button}
+                        >
+                            New Scene from Template
+                        </Button>
+                    </div>
+                </div>
+            </div>
```

Comment:
Can we load this dynamically if a prop (arbitrarily called data) is null / empty? Not exactly sure what the structure of the data prop will be, but null should be a safe thing to check against.

Reply:
Thinking a bit more, `scenes` would be a better name and then on the call that goes to this page, we can pass in scenario.scenes, which will default to an empty array if no scenes. so the conditional rendering should be based on `scenes !== []`.

Reply:
Was planning on doing this for the next ticket, unless you would like it implemented now.

Reply:
Sure next ticket is fine. As long as you are aware